### PR TITLE
add seder to transportPPLQueryResponse

### PR DESCRIPTION
**Pull request by zane-neo**
### Description
add seder to transportPPLQueryResponse
 
### Issues Resolved
add seder to transportPPLQueryResponse
 
### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass, including unit test, integration test and doctest
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
  - [ ] New functionality has user manual doc added
- [ ] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: plugin/src/main/java/org/opensearch/sql/plugin/transport/TransportPPLQueryResponse.java [hunk 1]
```diff
@@ -25,4 +31,22 @@ public TransportPPLQueryResponse(StreamInput in) throws IOException {
   public void writeTo(StreamOutput out) throws IOException {
     out.writeString(result);
   }
+
+  public static TransportPPLQueryResponse fromActionResponse(ActionResponse actionResponse) {
```
Comment (dai-chen) [inline plugin/src/main/java/org/opensearch/sql/plugin/transport/TransportPPLQueryResponse.java:35]: Could you add UT to show the use case of this method? In which case, we want to convert other ActionResponse to PPL response?

Comment (zane-neo) [reply plugin/src/main/java/org/opensearch/sql/plugin/transport/TransportPPLQueryResponse.java:35]: Added UT for this.

**Review by dai-chen — DISMISSED** [review]

**Comment by dai-chen** [comment]
Could you double check if the failed CI relevant or not? Thanks

**Comment by dai-chen** [comment]
Closing as new PR for `main` branch raised: https://github.com/opensearch-project/sql/pull/2452

**Pull request closed without merging**